src/shared/udev-util.c

```diff
@@ -108,8 +110,29 @@ int udev_parse_config_full(
         return 0;
 }
 
+static int device_new_from_dev_path(const char *devlink, sd_device **ret_device) {
+        struct stat st;
+        int r;
+
+        assert(devlink);
```

Comment:
i'd still like to the comment that says that ENOENT is the only error we don't log about (i mean, yeah, we do log about it debug level now, but that doesn't count. debug logging is always ok, and the rule i mentioned is actually about logging above debug level, not about logging at all

Reply:
No problem, added a comment